DataFormats/SiStripCluster/interface/SiStripCluster.h

```diff
@@ -51,21 +64,15 @@ class SiStripCluster {
    *  You can find the special meanings of values { 0, 254, 255} in section 3.4.1 of
    *  http://www.te.rl.ac.uk/esdg/cms-fed/firmware/Documents/FE_FPGA_Technical_Description.pdf
    */
-  auto size() const { return amplitudes_.size(); }
-  auto const* begin() const { return amplitudes_.data(); }
-  auto const* end() const { return begin() + size(); }
-  auto operator[](int i) const { return *(begin() + i); }
-  bool empty() const { return amplitudes_.empty(); }
+  uint8_t const* begin() const { return amplitudes_.data(); }
```

Comment:
you are undoing https://github.com/cms-sw/cmssw/pull/36852.
Please don't

Reply:
Sorry Marco, can you double check that I properly reverted the changes please?